test/clone-ns.c

```diff
@@ -0,0 +1,17 @@
+#define _GNU_SOURCE
```

Comment:
TESTDATA is `test/testdata` , I think that's a better fit for this anyway and moving it there will fix the test as is I think